src/main/resources/com/atlassian/jira/cloud/jenkins/config/JiraCloudPluginConfig/config.groovy

```diff
@@ -22,6 +22,13 @@ f.section(title: "Jira Software Cloud Integration") {
                 addCaption: _("Add Jira Cloud Site"))
     }
 
+    f.entry(title: "Logging"){
+        f.checkbox(
+                title: "Activate debug logging",
+                field: instance.FIELD_NAME_DEBUG_LOGGING
+        )
+    }
+
```

Comment:
This adds a checkbox to the Jenkins config page to enable the debug mode.

Reply:
so, we're letting the customer enable debug mode?  Shouldn't this be done via FF?

Reply:
this is a plugin installed into their Jenkins server ... we can't really do a feature flag here